src/sidebar/components/tag-editor.js

```diff
@@ -1,31 +1,38 @@
 import { createElement } from 'preact';
-import { useMemo, useRef, useState } from 'preact/hooks';
+import { useRef, useState } from 'preact/hooks';
 import propTypes from 'prop-types';
 
 import { withServices } from '../util/service-context';
 
+import Datalist from './datalist';
```

Comment:
Re-reading this code makes me wonder whether `Datalist` is still the right name for this component. I don't think it is a completely drop-in substitute for `<datalist>` any more is it (at least, not without making changes to the component containing the input field?). Perhaps we ought to rename it `Listbox` (to match the a11y role name) or `AutocompleteList`.

Reply:
I like 
AutocompleteList